test/integration/index.spec.js

```diff
@@ -779,3 +779,184 @@ describe('/publications FOR ALL TABLES', () => {
     assert.equal(stillExists, false)
   })
 })
+
+describe('/triggers', () => {
+  const trigger = {
+      name: 'test_trigger',
+      table: 'public.users_audit',
```

Comment:
I see you have a fully qualified function in the triggers logic, but I think on all the other routes we use `table` and `schema`. I think that's a bit cleaner

Reply:
@kiwicopple sounds good, I'll go and make the changes.